src/library/scala/collection/GenIterableLike.scala

```diff
@@ -147,3 +147,13 @@ trait GenIterableLike[+A, +Repr] extends Any with GenTraversableLike[A, Repr] {
   def zipAll[B, A1 >: A, That](that: GenIterable[B], thisElem: A1, thatElem: B)(implicit bf: CBF[Repr, (A1, B), That]): That
 
 }
+
+// Workaround for Scaladoc @usecase position bug scala/issues#11865
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
🙈

Reply:
After the [fix](https://github.com/scala/scala/pull/8668 ) is in an we re-STARR this can go.  👀 🐵

Reply:
Fix in https://github.com/scala/scala/pull/8668, but we'll need the workaround until we re-STARR atop that.

Reply:
The fix was merged, the de-slash-forestation can take place